### PR TITLE
Adding Swedish and Bulgarian translations #7289

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://github.com/mattermost/mattermost-webapp/pull/7289

Updated:
- User's Guide > Settings > Account Settings > Display
   - Added Bulgarian and Swedish as supported languages

### CODE REVIEW

**Comment by cwarnermm** [comment]
@esethna In the User's Guide > Getting Started > User Mattermost Desktop App > App Options > Check Spelling, we document a list of languages that support spell check. I've updated this list to match the current product UI. This updated list does not currently include all 18 supported languages, nor does it include the two most recently added languages: Bulgarian and Swedish. Please confirm whether all supported languages, or a subset of languages, support spell check. We may need to update the product UI text as well, depending on final confirmation.

**Comment by cwarnermm** [comment]
@adamjclarkson In the Cloud Admin Guide > Configuration > Site Configuration > Localization, I've updated the documentation to note a total of 18 languages now available with the addition of Bulgarian and Swedish. I'm making an assumption that Mattermost Cloud supports all of the same languages as Mattermost self-managed instances. Please confirm.

File: source/help/settings/account-settings.rst [hunk 1]
```diff
@@ -216,6 +216,7 @@ Language
 
 Select what language Mattermost displays in the user interface. Options include:
 
+- български - Bulgarian
```
Comment (jasonblais) [inline source/help/settings/account-settings.rst:219]: Bulgarian should be listed below Turkish
image

**Review by jasonblais — CHANGES_REQUESTED** [review]

**Comment by jasonblais** [comment]
> I'm making an assumption that Mattermost Cloud supports all of the same languages as Mattermost self-managed instances. Please confirm.

@cwarnermm that is correct

File: source/help/settings/account-settings.rst (hunk 1 above)
Comment (cwarnermm) [reply source/help/settings/account-settings.rst:219]: @jasonblais - Thank you. To clarify, would you like to see Bulgarian listed after Turkish in all cases where we list language support throughout the doc set (minus the Admin Guide config setting cases where we list supported languages alphabetically by short name only)? Or would you like to see this change for this page only?

Comment (jasonblais) [reply source/help/settings/account-settings.rst:219]: Just this page, which is for account settings specifically - hence why it's good to keep the same order with the Account Settings observed in the product.

In other pages, we resort to mostly alphabetical order.

File: source/overview/product.rst [hunk 2]
```diff
@@ -40,7 +40,7 @@ Features include:
 - Extensive integration support via `webhooks, APIs, drivers <https://docs.mattermost.com/guides/integration.html>`__ and `third party extensions <https://about.mattermost.com/default-app-directory/>`__
 - Easily scalable to dozens of users per team
 - New features and improvements released every two months
```
Comment (justinegeffen) [inline source/overview/product.rst:42]:
```suggestion
- New features and improvements released every month
```

File: source/help/apps/desktop-guide.rst [hunk 3]
```diff
@@ -64,7 +64,7 @@ Start app on login (Windows, Linux only)
     This setting is enabled by default.
 
 Check spelling
-    Highlight misspelled words in your messages. Available for English, French, German, Spanish, and Dutch. To change the spelling language, right-click inside a message box and navigate to **Spelling Languages** in the context menu.
+    Highlight misspelled words in your messages. Highlight misspelled words in your messages. Available for English, French, German, Portuguese, Russian, Ukrainian, Spanish, and Dutch. To change the spelling language, right-click inside a message box and navigate to **Spelling Languages** in the context menu.
```
[the remark this answers is not in the document]
Comment (cwarnermm) [reply source/help/apps/desktop-guide.rst:67]: Great catch. Whoops!

File: source/help/settings/account-settings.rst (hunk 1 above)
Comment (cwarnermm) [reply source/help/settings/account-settings.rst]: Thanks, @jasonblais. Updated.

**Review by jasonblais — APPROVED** [review]

**Review by justinegeffen — APPROVED** [review]

**Review by esethna — APPROVED** [review]

**Review by wiersgallak — APPROVED** [review]

**Pull request merged**